caravel/bin/caravel

```diff
@@ -125,6 +126,67 @@ def refresh_druid():
             "[" + cluster.cluster_name + "]")
     session.commit()
 
+@manager.option(
+    '-p', '--prefix', default="",
+    help="Sync Table Prefix")
+@manager.option(
+    '-d', '--database', default=None,
+    help="Specifies database (use caravel db name, not real db name)")
+def synctable(prefix, database):
+    ''' Sync DB Table with Caravel Table'''
+    print("")
+
+    if (prefix == "" and database is None) or (database == 'main' and prefix == ""):
```

Comment:
because if you use cmd to load examples, the db name: main
and the main db is same as Caravel core db